docs/gtfs/Gtfs_Instance_class_diagram.md

```diff
@@ -0,0 +1,25 @@
+# GtfsInstance
```

Comment:
Can't render diagram either with Quarto or RStudio. Am I missing dependencies?

Reply:
Ah, I see. Apologies I've not intended for use with quarto and I should have made this more clear. GitHub-flavoured markdown will render the mermaid diagram, the way I intended for this to be assessed would be within the [GitHub UI](https://github.com/datasciencecampus/transport-network-performance/blob/21-gtfs_instance-class-diagram/docs/gtfs/Gtfs_Instance_class_diagram.md).